Todo_CLI/todoCLI.js

```diff
@@ -0,0 +1,73 @@
+const readline = require('readline');
+const rl = readline.createInterface({
+	input: process.stdin,
+	output: process.stdout,
+});
+let list = []
```

Comment:
use `const` to declare, and name as constants, as new values will never be assigned
eg. `const LIST = []`